rust/kernel/static_call.rs

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: GPL-2.0
+
+// Copyright (C) 2024 Google LLC.
+
+//! Logic for static calls.
+
+#[macro_export]
+#[doc(hidden)]
+macro_rules! to_underscore {
+    ($arg:expr) => {
+        _
+    };
+}
+
+/// Imports the given static call.
+///
+/// To [`static_call!`] a function, you must import it into the local scope using this macro.
+///
+/// [`static_call!`]: macro@static_call
+#[macro_export]
+macro_rules! use_static_call {
+    ($name:ident) => {
+        $crate::macros::paste! { use $crate::bindings:: [<__SCT__ $name >]; }
+        $crate::macros::paste! { use $crate::bindings:: [<__SCK__ $name >]; }
```

Comment:
Maybe I'm missing something, but can you just put `use`s inside `_static_call!` body?